src/main/java/bitmap/transformer/App.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package bitmap.transformer;
+
+import java.io.IOException;
+
+public class App {
+    public String getGreeting() {
+        return "Hello world.";
+    }
+
+        public static void main(String[] args) {
+            Bitmap bmp = new Bitmap(args[0], args[1]);
+            bmp.greyScale();
+            bmp.border();
+            bmp.pixiLate();
```

Comment:
To complete implementing the CLI, put these method calls to your transformations inside 'if/else if' statements checking it with `args[2]`.  Like: `if(args[2].equals("greyScale") { bmp.greyScale( ) }`